plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/clients/FeatureDevClient.kt

```diff
@@ -148,6 +148,7 @@ class FeatureDevClient(private val project: Project) {
     suspend fun exportTaskAssistResultArchive(conversationId: String): MutableList<ByteArray> = amazonQStreamingClient.exportResultArchive(
         conversationId,
         ExportIntent.TASK_ASSIST,
+        null,
```

Comment:
Is this null intentional?

Reply:
Yeah, the `exportContext` field is only used by code transformation.